src/rocketData.py

```diff
@@ -172,9 +172,9 @@ def print_json_data(self):
         print(jsonData)
     
     #TODO: convert json data to csv format 
-    def convert_to_csv(self)
+    def convert_to_csv(self):
```

Comment:
This will cause merge conflicts. I know its wrong but its been addressed in a different branch
```suggestion
    def convert_to_csv(self)
```